templates/solutions/secure-open-source/index.html

```diff
@@ -19,9 +19,9 @@
     <section class="p-suru--25-75">
       <div class="row--25-75">
         <div class="col">
-          <h1>Standardise your software supply chain with secure open source</h1>
+          <h1>Secure your open source software supply chain</h1>
```

Comment:
The formatting here seems to be slightly off; the title in the figma doc is not as heavy

Reply:
![image](https://github.com/canonical/canonical.com/assets/27333173/a2963d38-b3b6-4cac-bbee-14281b5bfcf7)
vs 
![image](https://github.com/canonical/canonical.com/assets/27333173/a113c5cf-be37-4f16-b011-10ff527fba80)

Reply:
The design mentions it to be H1, which is the case.
Also there are two design in the figma doc.